components/modal/modal.css

```diff
@@ -0,0 +1,39 @@
+.modal-overlay {
+    position:fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0,0,0,0.5);
+    display: grid;
+    place-items: center;
+    visibility: hidden;
+    z-index: -10; 
+}
+
+.open-modal {
+    visibility: visible;
+    z-index: 10;
```

Comment:
Just curious to know why you have used large z-index:10? It could have been achieved by z-index:1 or 2.

Reply:
Yes I can use small z-index. It has become a habit to use large z-index for overlay. I will improve on this for next projects.